IdentityServer/v7/docs/content/diagnostics/otel/metrics.md

```diff
@@ -7,66 +7,74 @@ weight: 10
 (added in v7.0)
 
 The metric counters are designed to provide a high level overview. They are designed to not contain any
-sensitive information. The counters may contain tags to indicate the source of the data.
+sensitive information. The counters may contain tags to indicate the source of the data. Metrics are
+intended to be used to provide overall health metrics and are typically used to show graphs on
+a dashboard or to set up monitoring rules. If the metrics indicates that something has happened,
+the traces and logs are used to find out the reason. Open Telemetry monitoring tools might have
+provide features to find the traces and logs corresponding to certain metrics.
+
+IdentityServer emits metrics from the IdentityServer library. Our quick start for the UI also
+[contains metrics](#metrics-in-the-ui) that can be used as a starting point for monitoring UI events.
 
 ## High level Metrics
 These metrics are instrumented by the IdentityServer middleware and services and are
 intended to describe the overall usage and health of the system. They could provide the
-starting point for building a metrics dashboard.
+starting point for building a metrics dashboard. The high level metrics are enabled through
+the `Duende.IdentityServer.Telemetry.ServiceName` meter.
 
-#### Telemetry.Metrics.Counters.Failure/Success
-Counter names: *failure*/*success*
+#### Telemetry.Metrics.Counters.Operation
+Counter name: *tokenservice.operation*
 
-Aggregated counter of failed and successful operations. Note that it is expected to have failures during normal
-operation of IdentityServer. The failure/success ratio can be used as a very high level health metric.
+Aggregated counter of failed and successful operations. Note that it is expected to have some failures during
+normal operation of IdentityServer. There should be no internal_errors though, they indicate that there are
+unhandled exceptions. The failure/success ratio can be used as a very high level health metric.
```

Comment:
```suggestion
unhandled exceptions. The error/success ratio can be used as a very high level health metric.
```

(Since we don't have a failure event anymore, and the result will either be success or error)